### PR TITLE
don't use `require` to get functions from `draft-js/lib`

### PR DESCRIPTION
**Pull request by camille-hdl**
Hello,

This PR simply uses the standard `import` statement instead of `require()` to access `draft-js` exports.  

I encountered a problem in a project build with Rollup with code-splitting, esmodules and dynamic imports where using `require()` fails, this change solves it.

Your lib is very helpful to deal with copy/paste, thank you for your work!

* no API change
* no code added or removed
* tests ✅

### CODE REVIEW

**Comment by thibaudcolas** [comment]
Hey @camille-hdl, thanks for submitting this.

I can’t remember why I used `require` for those imports to be honest! If the tests do pass then this should be a pretty easy change to get in.

I see there are CI failures at the moment, if they are indeed related to those changes then I might need to look into why this confuses Flow. I’ll have a look as soon as I can.

File: src/lib/api/copypaste.js [hunk 1]
```diff
@@ -1,17 +1,17 @@
 // @flow
+import getContentStateFragment from "draft-js/lib/getContentStateFragment";
+import getDraftEditorSelection from "draft-js/lib/getDraftEditorSelection";
```
Comment (thibaudcolas) [inline src/lib/api/copypaste.js]: Looking good, I just had to do a bit more config to get rid of the Flow errors.

**Comment by thibaudcolas** [comment]
:tada: This pull request is included in v0.4.5, available on npm: [draftjs-conductor@0.4.5](https://www.npmjs.com/package/draftjs-conductor).

<p align="right">
Generated by :package::rocket: <a href="https://github.com/semantic-release/semantic-release">semantic-release</a>
</p>

**Comment by camille-hdl** [comment]
Great! thank you for your work @thibaudcolas

**Pull request merged**